.drone.yml

```diff
@@ -30,6 +30,7 @@ pipeline:
       - php occ config:system:set trusted_domains 1 --value=server
       - php occ config:system:set trusted_domains 2 --value=federated
       - php occ log:manage --level 0
+      - php occ config:system:set --value true --type boolean integrity.check.disabled
```

Comment:
@individual-it we will need to do similar in `password_policy` `.drone.yml` - it also sometimes sees the integrity check message when examining UI notifications in webUI tests.

Reply:
@phil-davis https://github.com/owncloud/password_policy/pull/213